events/listener.go

```diff
@@ -33,8 +35,15 @@ type EventRouter struct {
 	eventStream   *websocket.Conn
 	resourceName  string
 	pingConfig    PingConfig
+	eventLocker   EventLocker
 }
 
+type EventLocker func(event *Event) locks.Locker
+
+func nopLocker(_ *Event) locks.Locker { return locks.NopLocker() }
```

Comment:
it's not used in the method, so doesn't need to be named. I can even drop the underscore (just found out)